### PR TITLE
gettingstarted.rst

### PR DESCRIPTION
**Pull request by i0bs**
## Changes

- [X] I checked this pull request runs on `Python 3.6.X`.
- [ ] This fixes something in [Issues](https://github.com/eunwoo1104/discord-py-slash-command/issues).
    - Issue:
- [X] This adds something new.
- [ ] There is/are breaking change(s).
- [X] (If required) Document string is updated/added.
- [X] This changes anything except python code. (README, docs, etc.)

## About this pull request

This is a new page for the docs that allows you to learn more than the quickstart "basics" that are presented. It also adds a small update to the "quickstart.rst" and "index.rst" pages to show this page as well now.

### CODE REVIEW

File: docs/gettingstarted.rst [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    description="this returns the bot latency",
+    options=[manage_commands.create_option(
+      name = "argOne",
+      description = "description of first argument",
+      option_type = 3, # STRING type,
+      required = True
+    )],
+    guild_ids=guild_ids
+  )
+  async def _test(ctx, argOne: str):
```
Comment (MrSpinne) [inline docs/gettingstarted.rst:80]: I would suggest using `arg_one` instead of `argOne` as it is pythons naming convention.

Comment (MrSpinne) [inline docs/gettingstarted.rst:80]:
```suggestion
  async def _test(ctx, arg_one: str):
```

File: docs/gettingstarted.rst [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+  @client.event
+  async def on_ready():
+      print("Ready!")
+
+  @slash.slash(
+    name="test",
+    description="this returns the bot latency",
+    options=[manage_commands.create_option(
+      name = "argOne",
```
Comment (MrSpinne) [inline docs/gettingstarted.rst:73]:
```suggestion
      name = "arg_one",
```

File: docs/gettingstarted.rst [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+      description = "description of first argument",
+      option_type = 3, # STRING type,
```
Comment (spinnerich) [inline docs/gettingstarted.rst]:
```suggestion
      option_type = "STRING", # We can pass the name of the type directly
```
May be more conveniert.

**Review by spinnerich — COMMENTED** [review]
Maybe we should pass the name of the type? I think it might be easier for new users to handle.
Btw great work 🎉

File: docs/gettingstarted.rst [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+The main changes that you need to know about are with the lines calling the import
+of ``manage_commands``, as well as the ``options = [] ...`` code within the ``@slash.slash()``
+context coroutine. This will now create a new option called "argOne" when shown for
+the slash command.
+
+.. _quickstart: https://discord-py-slash-command.readthedocs.io/en/latest/quickstart.html
+.. _ApplicationCommandOptionChoice: https://discord.com/developers/docs/interactions/slash-commands#applicationcommandoptionchoice
+.. _ApplicationCommandOption: https://discord.com/developers/docs/interactions/slash-commands#applicationcommandoption
+
+.. toctree::
```
Comment (eunwoo1104) [inline docs/gettingstarted.rst]: We don't actually need toctree and `Indices and tables` for extra docs page.
It is only necessary for the index page.

Comment (i0bs) [reply docs/gettingstarted.rst]: Ah okay, I will remove it then.

**Comment by eunwoo1104** [comment]
Merging again, and thank you again!

**Pull request merged**